### PR TITLE
Dala-3017 -  stlye searchbar

### PR DESCRIPTION
**Pull request by stefanoferrario-eo**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [ ] At least one test exists testing the new feature
  - [ ] If you have created new test files, make sure that they are included in a test container and actually run in the CI
- [ ] Documentation is updated as required
- [ ] The automated deployment is updated if required
- [ ] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [ ] IF there are changes done to the framework data models or to a database entity class, the following steps were completed in order
  - [ ] The version of main currently active on prod is deployed to a dev server with `Reset non-user related Docker Volumes & Re-populate` turned on
  - [ ] It's verified that the CD run is green
  - [ ] The data from prod is migrated using `./migrateData.sh dataland.com <SOURCE_API_KEY> <TARGET> <TARGET_API_KEY>` 
  - [ ] The feature branch is deployed to the same server with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on the dev server
- [x] ELSE, the new version is deployed to the dev server "dev1" using this branch
  - [x] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [x] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [x] It's verified that the CD run is green
  - [x] The new feature is manually used/tested/observed on dev server
- [ ] Make sure that pull requests in the development tools repository, which are connected to this pull requests, 
are merged

### CODE REVIEW

File: dataland-frontend/src/components/resources/newLandingPage/TheIntro.vue [hunk 1]
```diff
@@ -133,6 +186,32 @@ const aboutIntroSection = computed(() => {
...
+        position: absolute;
+        left: -8px;
+        top: 50%;
+        transform: translateY(-50%);
+        width: 16px;
+        height: 16px;
+        background-image: url(/static/icons/Arrow--right.svg);
+        transform: rotateY(180deg) translateY(-50%);
+      }
+    }
```
Comment (BenediktLaubmann) [inline dataland-frontend/src/components/resources/newLandingPage/TheIntro.vue:214]: can we recycle the same button from the AuthSection here or add this styling to the global scss button styles ?

File: dataland-frontend/src/components/resources/newLandingPage/TheIntro.vue [hunk 2]
```diff
@@ -173,19 +264,77 @@ const aboutIntroSection = computed(() => {
...
+    &:focus {
+      box-shadow: none !important;
+      &::placeholder {
+        color: var(--grey-tones-300);
+      }
+    }
+  }
+
+  &__autocomplete {
```
Comment (BenediktLaubmann) [inline dataland-frontend/src/components/resources/newLandingPage/TheIntro.vue:284]: right now this styling is just valid for the CompaniesOnlySearchBar in the newLandingPage. Lets have a quick call if we also want this for the other occurrences of the CompaniesOnlySearchBar.

File: dataland-frontend/src/components/resources/companiesOnlySearch/CompaniesOnlySearchBar.vue [hunk 3]
```diff
@@ -83,8 +88,30 @@ export default defineComponent({
...
+     * resets the inputFocused data property to false and emits a 'blur' event.
+     * It is designed to be triggered when the autocomplete input loses focus.
+     */
+    handleInputBlur() {
+      setTimeout(() => {
+        this.inputFocused = false;
+        this.$emit("blur");
+      }, 300);
+    },
+
```
Comment (BenediktLaubmann) [inline dataland-frontend/src/components/resources/companiesOnlySearch/CompaniesOnlySearchBar.vue:114]: why do we need these?

**Review by BenediktLaubmann — COMMENTED** [review]
please merge main :)

File: dataland-frontend/src/components/resources/newLandingPage/TheIntro.vue (hunk 1 above)
Comment (stefanoferrario-eo) [reply dataland-frontend/src/components/resources/newLandingPage/TheIntro.vue:214]: Yes, that would be ideal but in this ticket I have no more time. Please open a new ticket with all these optimisations.

File: dataland-frontend/src/components/resources/newLandingPage/TheIntro.vue (hunk 2 above)
Comment (stefanoferrario-eo) [reply dataland-frontend/src/components/resources/newLandingPage/TheIntro.vue:284]: Yes, that would be ideal but in this ticket I have no more time. Please open a new ticket with all these optimisations.

File: dataland-frontend/src/components/resources/companiesOnlySearch/CompaniesOnlySearchBar.vue (hunk 3 above)
Comment (stefanoferrario-eo) [reply dataland-frontend/src/components/resources/companiesOnlySearch/CompaniesOnlySearchBar.vue]: We don't. I removed them

File: dataland-frontend/src/components/resources/newLandingPage/TheIntro.vue (hunk 1 above)
Comment (BenediktLaubmann) [reply dataland-frontend/src/components/resources/newLandingPage/TheIntro.vue:216]: okay

File: dataland-frontend/src/components/resources/newLandingPage/TheIntro.vue (hunk 2 above)
Comment (BenediktLaubmann) [reply dataland-frontend/src/components/resources/newLandingPage/TheIntro.vue:286]: okay

**Pull request merged**